### ISSUE TITLE
Can use `class AComponent extends React.Component` in index.js instead of `React.createClass` ??

### ISSUE THREAD

**Issue by haohcraft** [odysseyscience/react-router-proxy-loader#24]
If not, is it because webpack itself does not support es6? Or somehow we need to apply babel on webpack?

Just curious.

**Comment by seanadkinson** [comment]
Yeah, all dependent projects would need to have babel running, and process their node_modules (which isn't typical).  Whenever you have shared libraries, you usually either ship the compiled code, or dumb it down like we do here.
